lib/screens/finder.rb

```diff
@@ -55,12 +55,12 @@ def ask_to_expand
   end
 
   def expand_contact(matched)
-    index = ask_for_contact - 1
+    index = ask_for_contact_index - 1
```

Comment:
Sure ;) But should the Finder _care_ about that fact?
It's really a question about who should be responsible to translate a "user ID" to a "domain data ID".